lib/upland_mobile_commons_rest/profiles.rb

```diff
@@ -1,8 +1,19 @@
 module UplandMobileCommonsRest
   class Profiles < Base
+    def base_path
+      'profile'
+    end
+
+    def get(phone_number)
+      resp = client.get_request("#{base_path}?phone_number=#{phone_number}")
+      resp.body['response']['profile']
```

Comment:
Does digging into the profile object this way end up meaning that we do not get the profile ID? If it is included could you put that in the spec?

Reply:
XML attribute gets correctly added as a key on output hash. Will add to spec.